### DISCUSSION TITLE
Waveform simulation give different result from internal mesh and tomo

### DISCUSSION THREAD

**Discussion by Halauwet** [SPECFEM/specfem3d#1581] [SPECFEM3D General]
I tried to simulate the waveform but it gives different result from the simulation using internal mesher (utm coordinates) and using external tomo (source model and 1D velocity model are the same). I have checked the visual mesh and solver result (distance from source to receiver), it gives the same result but the plot shows different waveform amplitude (attached - label "SF" for specfem compared to simulation result using fomosto pyrocko - label "P" ).

Attached are the data and mesh files that I use. Can anyone explain why this could be happening?
[data.zip](https://github.com/SPECFEM/specfem3d/files/10425291/data.zip)

**Comment by danielpeter** [comment]
hi,

there are multiple modeling differences in your setup attached:
1. the mesh has different element sizes and length dimensions. this will affect the resolution of the output (minimum period resolved).
2. STACEY_INSTEAD_OF_FREE_SURFACE: in one of your setups, this is turned on, in the other off.
3. CMTSOLUTION: you're using a half-duration 0.0. this will use a (quasi-)Heaviside source time function and produce Green's functions. given that your resolution due to point (1.) is different, the Green's function will be different. thus, you need to filter both outputs for comparisons.
4. the Stacey boundary is not perfect, it will produce numerial artefacts. again, due to different mesh proportions in (1.), the boundary reflections will come in at different times. better make sure that the portion of the seismogram you're interested in is not affected by these artificial reflections.

with these differences in modeling the simulation, you would expect to see different waveforms.

**Reply by Halauwet** [reply]
Thanks @danielpeter, I missed the STACEY_INSTEAD_OF_FREE_SURFACE setup.

It has solved the problem as I compare the waveform at a relatively low frequency (< 0.1 Hz).
